flink-connector-vitess-cdc/src/main/java/io/debezium/connector/vitess/connection/VitessReplicationConnection.java

```diff
@@ -170,12 +170,13 @@ public void onError(Throwable t) {
                                                     + "lastProcessedVgtid:%s, "
                                                     + "lastErrorVgtid:%s, "
                                                     + "eventHandlingMode: %s, "
-                                                    + "internalRestarts: %s"),
-                                    vgtid,
-                                    lastProcessedVgtid,
-                                    lastErrorVgtid,
-                                    config.getEventProcessingFailureHandlingMode().getValue(),
-                                    internalRestarts.get());
+                                                    + "internalRestarts: %s",
```

Comment:
do not fix it here, fix it in vitess-eof-exception branch and also rebase
I will show you how later :D